### PR TITLE
#151980417 Design and Implement Landing Page 

### PR DESCRIPTION
**Pull request by Ruqoyah**
#### What does this PR do?
Design and implement landing page for users

#### Description of Task to be completed?
- Designed mock up for landing page
- Choose 3 compatible colors
- Implement a creative landing page

#### How should this be manually tested?
npm start to start the application then launch it in the browser
 
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/151980417

### CODE REVIEW

**Comment by williamolojede** [comment]
@Ruqoyah @WillyWunderdog you guys will need to make a couple of changes to ur Procfile and package.json for the app to work successfully on heroku.
```
Procfile
web: ./node_modules/.bin/forever -m 5 ./server.js
```

``` json
package.json
...
"scripts": {
...
"heroku-postbuild": "npm run bower"
...
}
...
```

or better still pull my current `feature/151980414/login-auth-jwt` branch to have the current working heroku configuration cc @andela-moe

File: public/css/style.css [hunk 1]
```diff
@@ -1,1257 +1,339 @@
...
-   border-radius: 8px;
-   background: #252525;
-   padding: 5px 5px 5px 13px;
+.navbar-toggler {
+  color: white;
 }
 
-#first-word {
-   color: #7CE4E8;
+span {
```
Comment (codrex) [inline public/css/style.css]: Class selector would appropriate.

File: public/css/style.css [hunk 2]
```diff
@@ -1,1257 +1,339 @@
...
+  top: 0;
+  left: 0;
+  bottom: 0;
+  right: 0;
+  margin: auto;
+  height: 15px;
+  z-index: 20;
+}
+
+h1 {
```
Comment (codrex) [inline public/css/style.css]: class selector would more appropriate.

File: public/css/style.css [hunk 3]
```diff
@@ -1,1257 +1,339 @@
...
-
-@media (min-width: 1430px) {
-   .row {
-      margin: 0 0 -9px 0;
-   }
-
-   .wpb-logo-container {
-      margin-right: 0;
-   }
+w{
```
Comment (codrex) [inline public/css/style.css]: Please, I would love to know what w is doing?

File: public/css/style.css [hunk 4]
```diff
@@ -1,1257 +1,339 @@
...
-      padding-top: 70px;
-   }
+  .wrap {
+    height: 300px;
+  }
 
-   #ang-logo {
-      width: 37% !important;
-   }
+  h1 {
```
Comment (codrex) [inline public/css/style.css]: class selector would more appropriate.

**Pull request merged**